src/index.ts

```diff
@@ -397,57 +400,46 @@ function dontParse(paths: webpack.Module["noParse"]): webpack.Configuration {
   };
 }
 
-// https://webpack.js.org/configuration/dev-server/#devserver
+// https://www.npmjs.com/package/webpack-plugin-serve
 //
-// Note that HMR is enabled by default! That could be extracted to
-// another function in case it's not needed in all projects.
-function webpackDevServer(
-  options: WebpackDevServer.Configuration,
-): webpack.Configuration {
-  if (process.env.STORYBOOK) {
-    return {};
-  }
-
-  return {
-    devServer: {
-      hot: true,
-      headers: {
-        "Access-Control-Allow-Origin": "*",
-      },
-      ...options,
-    },
-    plugins: [new webpack.HotModuleReplacementPlugin()],
-  };
-}
-
-// This is the legacy option for React projects. It requires you to use
-// hot wrapper from react-hot-loader at the root of an app.
-function reactHotLoader(): webpack.Configuration {
+// Note that when using webpack-plugin-serve, you have to run
+// the process through regular webpack, not webpack-dev-server!
```

Comment:
is this comment still up to date, given that we dropped `webpack-dev-server` now?

Reply:
I improved the wording.